src/sentry/models/auditlogentry.py

```diff
@@ -87,6 +87,8 @@ class AuditLogEntryEvent(object):
 
     INTERNAL_INTEGRATION_ADD = 130
 
+    SENTRY_APP_INSTALLATION_TOKEN_CREATED = 135
```

Comment:
I would change this to be something like `INTEGRATION_ADD_TOKEN` and then you need to make sure you include this below i.e.
 `(AuditLogEntryEvent.INTEGRATION_ADD_TOKEN, 'internal_integration.token_add')`

```
elif self.event == AuditLogEntryEvent.INTEGRATION_ADD_TOKEN:
            return 'created a token for integration %s ' % (
                self.data['sentry_app'])
```